webpack.config.js

```diff
@@ -93,6 +93,19 @@ module.exports = [
         },
       ],
     },
+
+    resolve: {
+      alias: {
+        app: path.resolve(__dirname, './client/control/app.js'),
+        routes: path.resolve(__dirname, './client/control/routes.js'),
+        stores: path.resolve(__dirname, './client/control/stores'),
+        reducers: path.resolve(__dirname, './client/control/reducers'),
+        actions: path.resolve(__dirname, './client/control/actions'),
+        components: path.resolve(__dirname, './client/control/components'),
+        tests: path.resolve(__dirname, './client/control/tests'),
+        selfrepair: path.resolve(__dirname, './client/selfrepair'),
```

Comment:
I see. I had been swapping them out for easier use in the `control` directory, but you'd like it to point to the directories inside `client`.

I'll get to fixing this now.